setup-machine-arktos.sh

```diff
@@ -146,6 +142,26 @@ sudo pip3 install netaddr
 sudo pip3 install kopf
 sudo pip3 install PyYAML
 
+#####################
+
+echo Setup: Install Containerd
+
```

Comment:
Create a new shell script to replace containerd for better separation. 
Containerd replace is integration specific, should be separated from machine setup step

Reply:
Create a new file such as replace-containerd.sh, and move containerd related scripts to the new file

Reply:
I have added a separate shell script as per your suggestion to replace the containerd called replace-containerd.sh